pkg/cloudprovider/cloudprovider.go

```diff
@@ -57,3 +57,21 @@ func GetCloudConfig(pconfig providerconfigtypes.Config, kubeletVersion string) (
 
 	return "", errors.New("unknown cloud provider")
 }
+
+func KubeletCloudProviderName(cloudProvider providerconfigtypes.CloudProvider) (osmv1alpha1.CloudProvider, error) {
```

Comment:
What is the need for this?

Reply:
It is needed to specify the `cloud-provider` flag in the Kubelet.

Reply:
Don't we have a flag that specifies if the cloud provider is external or not?